src/test/java/com/petros/bibernate/session/context/PersistenceContextTest.java

```diff
@@ -57,8 +78,42 @@ void clear() {
     }
 
     @Test
-    @DisplayName("Testing entity snapshots")
+    @DisplayName("If entity has changed field it's added to snapshot")
     void testEntitySnapshot() {
```

Comment:
There is a lack of tests for entities with @OneToMany and @ManyToOne annotations.